packages/ia-components/components/theatres/with-youtube-spotify/audio-with-youtube-spotify-main.jsx

```diff
@@ -431,13 +436,18 @@ AudioPlayerWithYoutubeSpotify.defaultProps = {
   jwplayerPlaylist: null,
   playFullIAAudio: false,
   linerNotes: null,
+  userSignedIn: false,
+  albumMetadataStr: '',
+  albumMetadata: null,
```

Comment:
We know there are two new properties which is **albumMetadataStr** and **userSignedIn**, Just curious setting albumMetadata to null won't make anything wrong?

Reply:
hi neeraj, good question, no it doesn't actually affect anything.  all of these properties get hydrated at the same time so we should be good.  the linter was barking that `albumMetadata` didn't have a prop so i added one.